### DISCUSSION TITLE
Simplify date/datetime datatypes

### DISCUSSION THREAD

**Discussion by sundy-li** [databendlabs/databend#4909] [General]
Proposal:

1.  Union `Date`(`Date16`)    `Date32` to be `Date` type, which uses i32 as physical datatype.
2.   Union `DateTime` (`DateTime32`), `DateTime64` to be `DateTime` type, which uses i64 as physical datatype.

**Comment by bohutang** [comment]
For 2., in duckdb is timestamp:
https://duckdb.org/docs/sql/data_types/timestamp

In snowflake:
```
DATETIME is an alias for TIMESTAMP_NTZ.
```
